server/tests/subtests/realtime_test.rs

```diff
@@ -170,6 +170,110 @@ pub async fn remove_add_modify_vj_test(config: &ServerConfig) {
         assert_eq!(journeys_response.journeys.len(), 0);
     }
 
+    // let's now 'add' the removed base trip "matin" with flag SIGNIFICANT_DELAYS
+    // this should add back the trip, since the vehicle exists in base schedule
+    {
+        let realtime_message = create_disruption(
+            "matin",
+            date,
+            vec![
+                ("massy", date.and_hms(8, 0, 0)),
+                ("paris", date.and_hms(9, 0, 0)),
+            ],
+            kirin_proto::Alert_Effect::SIGNIFICANT_DELAYS,
+        );
+        crate::send_realtime_message_and_wait_until_reception(config, realtime_message).await;
+
+        // since nothing should be added, we should get
+        // no journey for the request on the realtime level
+        let journeys_response = crate::send_request_and_wait_for_response(
+            &config.requests_socket,
+            realtime_request.clone(),
+        )
+        .await;
+        assert_eq!(
+            first_section_vj_name(&journeys_response.journeys[0]),
+            "vehicle_journey:matin"
+        );
+    }
+}
+
+pub async fn remove_add_modify_new_vj_test(config: &ServerConfig) {
+    // the ntfs (in tests/a_small_ntfs) contains just one trip
+    // with a vehicle_journey named "matin"
+    // dparting from "massy" at 8h and arriving to "paris" at 9h
```

Comment:
There are probably other occurrences of that one:
```suggestion
    // departing from "massy" at 8h and arriving to "paris" at 9h
```